python/ngraph_bridge/__init__.in.py

```diff
@@ -105,21 +105,21 @@ def requested():
     })
 
 
-ngraph_bridge_lib.ngraph_is_enabled.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_list_backends.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_set_backend.argtypes = [ctypes.c_char_p]
-ngraph_bridge_lib.ngraph_set_backend.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_is_supported_backend.argtypes = [ctypes.c_char_p]
-ngraph_bridge_lib.ngraph_is_supported_backend.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_get_currently_set_backend_name.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_is_logging_placement.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_tf_version.restype = ctypes.c_char_p
-ngraph_bridge_lib.ngraph_lib_version.restype = ctypes.c_char_p
-ngraph_bridge_lib.ngraph_tf_cxx11_abi_flag.restype = ctypes.c_int
-ngraph_bridge_lib.ngraph_tf_is_grappler_enabled.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_tf_are_variables_enabled.restype = ctypes.c_bool
-ngraph_bridge_lib.ngraph_set_disabled_ops.argtypes = [ctypes.c_char_p]
-ngraph_bridge_lib.ngraph_get_disabled_ops.restype = ctypes.c_char_p
+ngraph_bridge_lib.NgraphIsEnabled.restype = ctypes.c_bool
+ngraph_bridge_lib.NgraphListBackends.restype = ctypes.c_bool
+ngraph_bridge_lib.NgraphSetBackend.argtypes = [ctypes.c_char_p]
+ngraph_bridge_lib.NgraphSetBackend.restype = ctypes.c_bool
+ngraph_bridge_lib.NgraphIsSupportedBackend.argtypes = [ctypes.c_char_p]
+ngraph_bridge_lib.NgraphIsSupportedBackend.restype = ctypes.c_bool
+ngraph_bridge_lib.NgraphGetCurrentlySetBackendName.restype = ctypes.c_bool
+ngraph_bridge_lib.NgraphIsLoggingPlacement.restype = ctypes.c_bool
+ngraph_bridge_lib.NgraphBridgeVersion.restype = ctypes.c_char_p
+ngraph_bridge_lib.NgraphLibVersion.restype = ctypes.c_char_p
+ngraph_bridge_lib.NgraphBridgeCXX11ABIFlag.restype = ctypes.c_int
+ngraph_bridge_lib.NgraphBridgeAreVariablesEnabled.restype = ctypes.c_bool
+ngraph_bridge_lib.NgraphBridgeAreVariablesEnabled.restype = ctypes.c_bool
```

Comment:
119 and 120 is same

Reply:
oops, will change it. one of them is grappler.